### PR TITLE
Fix doc comment in pp.mli

### PR DESCRIPTION
**Pull request by SkySkimmer**

### CODE REVIEW

File: lib/pp.mli [hunk 1]
```diff
@@ -10,30 +10,30 @@
...
+(**                                                                      *)
+(** That is to say, this:                                                *)
+(**                                                                      *)
+(** `hov [str "Term"; hov (pr_term t); str "is defined"]`                *)
+(**                                                                      *)
+(** is preferred to:                                                     *)
+(**                                                                      *)
+(** `hov (str "Term" ++ hov (pr_term t) ++ str "is defined")`            *)
+(**                                                                      *)
+(*************************************************************************)
```
Comment (ejgallego) [inline lib/pp.mli:36]: Alternatively we could just drop all the per-line `(**` and use a single `(** ... *)` block; we also need to update the syntax from "markdown" to "ocamldoc".

Comment (SkySkimmer) [reply lib/pp.mli]: I don't know ocamldoc syntax

File: lib/pp.mli [hunk 2]
```diff
@@ -10,30 +10,31 @@
...
-(*                                                                      *)
-(* `hov [str "Term"; hov (pr_term t); str "is defined"]`                *)
-(*                                                                      *)
-(* is preferred to:                                                     *)
-(*                                                                      *)
-(* `hov (str "Term" ++ hov (pr_term t) ++ str "is defined")`            *)
-(*                                                                      *)
-(************************************************************************)
+(**
+Pretty printing guidelines
```
Comment (ejgallego) [inline lib/pp.mli:14]: Should be `{5 Pretty-printing guildelines}`

File: lib/pp.mli [hunk 3]
```diff
@@ -10,30 +10,31 @@
...
+
+`Pp.t` is the main pretty printing document type
```
Comment (ejgallego) [inline lib/pp.mli:16]: Should be `[Pp.t]`

File: lib/pp.mli (hunk 1 above)
Comment (ejgallego) [reply lib/pp.mli]: Mainly we need to turn quotes into brackets, see https://caml.inria.fr/pub/docs/manual-ocaml/ocamldoc.html#ss:ocamldoc-formatting

**Comment by ejgallego** [comment]
@SkySkimmer can you push these nits? Thanks! [I could push them myself but then I'd block the merge by myself]

**Comment by SkySkimmer** [comment]
updated

**Review by ejgallego — APPROVED** [review]
Thanks

**Comment by ejgallego** [comment]
@coqbot: merge now

**Pull request merged**